synapse/logging/formatter.py

```diff
@@ -28,10 +30,14 @@ class LogFormatter(logging.Formatter):
     where it was caught are logged).
     """
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
```

Comment:
Removed this since it did literally nothing other than create a new stack frame!